labview-interop/src/memory.rs

```diff
@@ -141,8 +143,32 @@ impl<T: ?Sized> DerefMut for UHandle<T> {
     }
 }
 
+//TODO test
 #[cfg(feature = "link")]
-impl<T: ?Sized> UHandle<T> {
+impl<'a, T: ?Sized> Borrow<UHandle<'a, T>> for LvOwned<'a, T> {
+    fn borrow(&self) -> &UHandle<'a, T> {
+        self.deref()
+    }
+}
+
+//TODO test
+#[cfg(feature = "link")]
+impl<'a, T: ?Sized> ToOwned for UHandle<'a, T> {
```

Comment:
This implementation should be in the lvowned submodule. Grouping it with LvOwned makes sense and I put it in a submodule so we can have the single feature flag for link to cover all of the contents.